### ISSUE TITLE
서브 테스크 이슈템플릿 적용안됨

### ISSUE THREAD

**Issue by yongseokleecom** [yona-projects/yona#419]
버전: H2 1.10.0

이슈템플릿을 사용중인데 "새 이슈" 를 사용할때는 템플릿이 적용되나
"새 서브 태스크" 로 등록하면 템플릿이 적용되지 않습니다.
확인 부탁드립니다.

**Comment by doortts** [comment]
네. 현재는 의도적으로 서브 태스크에는 템플릿이 적용되지 않도록 설정되어 있습니다.

배경설명
--
초반에는 서브 태스크에도 템플릿을 적용했었는데 서브 태스크는 부모 태스크 의존적인 TODO ITEM 적인 성격이 많아서 프로젝트의 이슈 템플릿이 무의미한 경우가 많았었습니다. 

고민
---
그래서 템플릿 옵션을 제공할까도 살짝 고민했는데 사용성을 좀 더 살펴보고 그 때 보완하는 작업을 진행하기로 정해서 현재처럼 되었습니다.

**Comment by yongseokleecom** [comment]
@doortts 
그래서 버튼 이름이 "새 서브 이슈" 아니고 "새 서브 태스크" 인거군요.
내부에서 사용할때 서브 이슈 개념으로 사용하다보니 필요한데 옵션 제공가능성이 있을가요?

**Comment by doortts** [comment]
네. 다음 버전에서는 몇 가지 실험적인 기능들(멀티 템플릿, 이슈 생성시 추가 옵션, 본문 Checklist 등)이 포함될 예정입니다.
